src/main/java/com/cburch/logisim/prefs/FpgaBoards.java

```diff
@@ -184,6 +184,7 @@ public void actionPerformed(ActionEvent e) {
   private JScrollPane boardPane;
   private JList<String> boardNamesList;
   private JButton addButton;
+  private JLabel ExtBoardPanelTitl;
```

Comment:
Variables should start with a lowercase letter.

Reply:
Normally, this is corrected